src/styles/_common.scss

```diff
@@ -1 +1,17 @@
-$card-background-color: #f5f5f5;
\ No newline at end of file
+$card-background-color: #f5f5f5;
+
+
+.cta--center--position{
+    margin-left: 40%;
+    margin-top: 157px;
+    margin-bottom: 127px;
+
+}
+
+.carousel--center--position{
+    top: 846px;
```

Comment:
Refrain using absolute position and specific values on it. Used our spacing css. Thank you.